.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -23,12 +23,12 @@ body:
   - type: textarea
     id: result
     attributes:
-      label: What happened?
+      label: Intended result and actual result
```

Comment:
Please match Element web on this, the label should be `Outcome`

Reply:
Done.